grpc-api/src/main/java/io/deephaven/grpc_api/arrow/FlightServiceGrpcImpl.java

```diff
@@ -460,15 +462,14 @@ private void sealAndExport() {
                 }
 
                 // no more changes allowed; this is officially static content
-                resultTable.sealTable();
-                resultExportBuilder.submit(() -> {
+                resultTable.sealTable(() -> resultExportBuilder.submit(() -> {
                     // transfer ownership to submit's liveness scope, drop our extra reference
                     resultTable.manageWithCurrentScope();
                     resultTable.dropReference();
+                    observer.onCompleted();
```

Comment:
wrap this up so we can still export if network dies at this point? the table still exists and is sealed and exported, and the user will hopefully be back shortly